app/views/layouts/_custom_flash.html.erb

```diff
@@ -0,0 +1,4 @@
+<% flash.each do |name, message| %>
+  <%= content_tag(:div, message, :id => "flash_#{name}") %>
+  <%= link_to "CLICK HERE TO RESUBMIT YOUR DONATION", candidates_path %>
```

Comment:
Ok so just redirecting to the candidates page -- maybe the undo action was too ambitious. Feel like it should be doable but I agree lets table it for later.